.profile

```diff
@@ -4,3 +4,4 @@
 ##
 
 export https_proxy=$egress_proxy
+export NEW_RELIC_PROXY_HOST=$egress_proxy
```

Comment:
This made me realize that we could use some docs about the interplay between various files at deploy time. I opened https://github.com/GSA/notifications-api/issues/169 to track that.